content/influxdb/v2.6/reference/syntax/delete-predicate.md

```diff
@@ -32,6 +32,22 @@ Use [logical operators](#logical-operators) to combine two or more predicate exp
 key1="value1" AND key2="value"
 ```
 
+{{% warn %}}
+#### Predicates with special characters or keywords
+Sometimes your predicate may contain special characters or keywords. To make sure they are properly parsed, they will need to be escape quoted. See below for several examples:
+
+```
+// Escaped due to the "-"
+"_measurement=\"example-dash\""
+```
+
+```
+// Escaped because "name" is a keyword
+"_measurement=example and \"name\"=predicate"
+```
```

Comment:
~~~suggestion
```js
// Escaped due to the "-"
"_measurement=\"example-dash\""

// Escaped because "name" is a keyword
"_measurement=example and \"name\"=predicate"
```
~~~